CDS_CarrotJob/CDS_CarrotJob/Presentation/Home/Cell/PartTimeJobCollectionViewCell.swift

```diff
@@ -0,0 +1,200 @@
+        
+        advertisingButton.snp.makeConstraints {
+            $0.top.equalTo(jobLocationLabel)
+            $0.leading.equalTo(jobLocationLabel.snp.trailing).offset(5)
+            $0.width.equalTo(40)
+            $0.height.equalTo(18)
+        }
+        
+        jobImage.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(14)
+            $0.trailing.equalToSuperview().inset(16)
+            $0.width.height.equalTo(70)
+        }
+        
+        jobDayWeekLabel.snp.makeConstraints {
+            $0.bottom.equalToSuperview().inset(15)
+            $0.leading.equalToSuperview().inset(16)
+            $0.height.equalTo(20)
+        }
+        
+        lineView.snp.makeConstraints {
+            $0.bottom.equalToSuperview().inset(19)
+            $0.leading.equalTo(jobDayWeekLabel.snp.trailing).offset(8)
+            $0.width.equalTo(1)
+            $0.height.equalTo(11)
+        }
+        
+        jobTimeLabel.snp.makeConstraints {
+            $0.bottom.equalTo(jobDayWeekLabel)
+            $0.leading.equalTo(lineView.snp.trailing).offset(8)
+            $0.height.equalTo(20)
+        }
+        
+        reviewCountLabel.snp.makeConstraints {
+            $0.bottom.equalTo(jobDayWeekLabel.snp.top).offset(-5)
+            $0.leading.equalTo(jobDayWeekLabel)
+            $0.height.equalTo(20)
+        }
+        
+        contactStatusView.snp.makeConstraints {
+            $0.bottom.equalTo(jobDayWeekLabel.snp.top).offset(-2)
+            $0.leading.equalTo(reviewCountLabel.snp.trailing).offset(8)
+            $0.width.equalTo(90)
+            $0.height.equalTo(24)
+        }
+        
+        distanceTimeStatusView.snp.makeConstraints {
+            $0.bottom.equalTo(contactStatusView)
+            $0.leading.equalTo(contactStatusView.snp.trailing).offset(8)
+            $0.width.equalTo(72)
+            $0.height.equalTo(24)
+        }
+        
+        jobHourlyWageLabel.snp.makeConstraints {
+            $0.bottom.equalToSuperview().inset(16)
+            $0.trailing.equalToSuperview().inset(16)
+        }
+        
+        underLineView.snp.makeConstraints {
+            $0.bottom.equalToSuperview()
+            $0.leading.trailing.equalToSuperview().inset(17)
+            $0.height.equalTo(1)
+        }
+    }
+    
+    // MARK: - Methods
+    
+    func setDataBind(serverModel: PartTimeServerModel, dummyModel: PartTimeJobModel) {
+        jobNameLabel.text = serverModel.jobName
+        jobLocationLabel.text = dummyModel.jobLocation
+        reviewCountLabel.text = dummyModel.reviewCount
+        jobDayWeekLabel.text = dummyModel.jobDayWeek
+        jobTimeLabel.text = dummyModel.jobTime
+        guard let url = URL(string: serverModel.jobImage) else { return }
+        jobImage.kf.setImage(with: url)
+        var hourlyWage = String(serverModel.jobHourlyWage)
+        if hourlyWage.count > 3 {
+            hourlyWage.insert(",", at: hourlyWage.index(hourlyWage.endIndex, offsetBy: -3))
+        }
```

Comment:
p3
이 자릿수 comma 쓰는건 따로, Int Extension 으로 빼서 씁시다! ㅎㅎ (제가 만들게요)
insert 의 경우, 사실 여기선 큰 상관이 없겠지만 시간복잡도가 O(N) 이라 안쓰게 되더라구여 ㅜㅠ

지금은! 
```suggestion
        }
        let numberFormatter = NumberFormatter()
        numberFormatter.numberStyle = .decimal
        var hourlyWage = serverModel.jobHourlyWage
        if let wage = numberFormatter.string(for: hourlyWage) {
            jobHourlyWageLabel.text = "시급 " + "\(wage)" + "원"
        }
```

이렇게 바꾸면 어떨까요?!

Reply:
오 !!!!! 세미나에서 string에 콤마 넣었던 게 기억나서 그대로 써봤는데 시간복잡도 문제가 있었군뇨... 나중에 풀 받구 바로 변경하겠슴당 ㅎㅎ